core-services/image-mirroring/openshift/mapping_origin_4_3

```diff
@@ -155,6 +155,8 @@ registry.svc.ci.openshift.org/origin/4.3:metering-reporting-operator quay.io/ope
 registry.svc.ci.openshift.org/origin/4.3:migrator quay.io/openshift/origin-migrator:4.3 quay.io/openshift/origin-migrator:4.3.0
 registry.svc.ci.openshift.org/origin/4.3:multus-admission-controller quay.io/openshift/origin-multus-admission-controller:4.3 quay.io/openshift/origin-multus-admission-controller:4.3.0
 registry.svc.ci.openshift.org/origin/4.3:multus-cni quay.io/openshift/origin-multus-cni:4.3 quay.io/openshift/origin-multus-cni:4.3.0
+registry.svc.ci.openshift.org/origin/4.3:multus-route-override-cni quay.io/openshift/origin-multus-route-override-cni:4.3 quay.io/openshift/origin-multus-route-override-cni:4.3.0
```

Comment:
I don't see a build definition for this in 4.3 - is that intentional?  If so, remove this.